source/sayAllHandler.py

```diff
@@ -49,7 +49,7 @@ def readObjects(obj):
 	_startGenerator(readObjectsHelper_generator(obj))

```

Comment:
All changes to this file are going to conflict with #7599. Please revert these as well. They will be addressed during the Python 3 transition.

Reply:
I've reverted the changes.